lib/va_profile/disability/service.rb

```diff
@@ -58,19 +54,19 @@ def identity_path
       private
 
       def edipi_present!
-        raise 'User does not have a valid edipi' if @edipi.blank?
+        raise 'User does not have a valid edipi' if @user&.edipi.blank?
       end
 
       def edipi_with_aaid
         "#{edipi_id}#{aaid}"
       end
 
       def edipi_id
-        @edipi.presence
+        @user&.edipi.presence
```

Comment:
This should also be  @edipi. The edipi is sent in the initial request to get the user.